src/devices/Ina226/Ina226Enum.cs

```diff
@@ -0,0 +1,249 @@
+        Time332us = 0b00000000_10000000,
+
+        /// Conversion time bits
+        Time588us = 0b00000000_11000000,
+
+        /// Conversion time bits
+        Time1100us = 0b00000001_00000000,
+
+        /// Conversion time bits
+        Time2116us = 0b00000001_01000000,
+
+        /// Conversion time bits
+        Time4156us = 0b00000001_10000000,
+
+        /// Conversion time bits
+        Time8244us = 0b00000001_11000000
+    }
+
+    /// <summary>
+    /// An enumeration representing flags and masks used for the alert enable register on the INA226
+    /// </summary>
+    [Flags]
+    internal enum Ina226AlertEnableFlags : ushort
+    {
+        ShuntOV = 0b10000000_00000000,
+        ShuntUV = 0b01000000_00000000,
+        BusOV = 0b00100000_00000000,
+        BusUV = 0b00010000_00000000,
+        PowerOL = 0b00001000_00000000,
+        ConvReady = 0b00000100_00000000,
+        AlertPolarity = 0b00000000_00000010,
+        AlertLatchEn = 0b00000000_00000001
+    }
+
+    /// <summary>
+    /// An enumeration representing the different alert trigger modes available on the INA226
+    /// </summary>
+    public enum Ina226AlertMode : ushort
+    {
+        /// <summary>Mode to trigger ShuntOverVoltage alert if shunt register exceeds alert limit register</summary>
+        ShuntOverVoltage = 0x8000,  // This is in hex because the binary equivalent causes an out of limit exception for some reason...
+
+        /// <summary>Mode to trigger ShuntUnderVoltage alert if shunt register is below alert limit register</summary>
+        ShuntUnderVoltage = 0b01000000_00000000,
+
+        /// <summary>Mode to trigger BusOverVoltage alert if bus register exceeds alert limit register</summary>
+        BusOverVoltage = 0b00100000_00000000,
+
+        /// <summary>Mode to trigger BusUnderVoltage alert if bus register is below alert limit register</summary>
+        BusUnderVoltage = 0b00010000_00000000,
+
+        /// <summary>Mode to trigger PowerOverLimit alert if Power register exceeds alert limit register</summary>
+        PowerOverLimit = 0b00001000_00000000,
+
+        /// <summary>Mode to trigger ConversionReady alert when the current conversion has been finished</summary>
+        ConversionReady = 0b00000100_00000000
+    }
+
+    /// <summary>
+    /// An enumeration representing the flags and masks used to configure the alert register
+    /// </summary>
+    public enum Ina226AlertMask : ushort
+    {
+        /// Mask used for the alert mode bit(s) to be set
+        AlertModeMask = 0b11111100_00000000,
+
+        /// Mask for alert polarity bit(s) to be set
+        AlertPolarityMask = 0b00000000_00000010,
+
+        /// Mask for alert latch bit(s) to be set
+        AlertLatchMask = 0b00000000_00000001,
+
+        /// Mask for alert math over flow flag to be read
+        AlertMathOverflowFlag = 0b00000000_00000100,
+
+        /// Mask for conversion ready flag to be read
+        AlertConvReadyFlag = 0b00000000_00001000,
```

Comment:
Conv => Conversion

note: for internal/private stuff I don't care, for public stuff it should not use abbreviations